server/content-element/content-element.controller.js

```diff
@@ -38,7 +38,10 @@ function patch({ repository, user, body, params: { elementId } }, res) {
   const context = { userId: user.id, repository };
   return ContentElement.findByPk(elementId, { paranoid })
     .then(asset => asset || createError(NOT_FOUND, 'Element not found'))
-    .then(asset => asset.update(data, { context }))
+    .then(asset => {
+      asset.setDataValue('deletedAt', data.deletedAt);
```

Comment:
@hrvojevu 
It should be only possible to restore assets if deleted (please make it a bit more specific).
Thx!